app/controllers/crime_applications_controller.rb

```diff
@@ -41,7 +64,15 @@ def sorting_params
     { helpers.sort_by => helpers.sort_direction }
   end
 
+  def new_application_params
+    params.fetch(:start_is_cifc_form, {}).permit(:is_cifc)
+  end
+
   def log_context
     LogContext.new(current_provider: current_provider, ip_address: request.remote_ip)
   end
+
+  def cifc?
```

Comment:
Can we use @form_object.is_cifc rather than params?

Reply:
@mseedat-moj perhaps something to refactor on monday